src/main/scala/org/broadinstitute/dsde/rawls/webservice/MethodConfigApiService.scala

```diff
@@ -0,0 +1,209 @@
+    path("workspaces" / Segment / Segment / "methodconfigs" ) { (workspaceNamespace, workspaceName) =>
+      get {
+        requestContext => perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor),
+          WorkspaceService.ListMethodConfigurations(workspaceNamespace, workspaceName))
+      }
+    }
+  }
+
+  @Path("/{configNamespace}/{configName}")
+  @ApiOperation(value = "delete method configuration in a workspace",
+    nickname = "delete method configuration",
+    httpMethod = "Delete")
+  @ApiImplicitParams(Array(
+    new ApiImplicitParam(name = "workspaceNamespace", required = true, dataType = "string", paramType = "path", value = "Workspace Namespace"),
+    new ApiImplicitParam(name = "workspaceName", required = true, dataType = "string", paramType = "path", value = "Workspace Name"),
+    new ApiImplicitParam(name = "methodConfigurationNamespace", required = true, dataType = "string", paramType = "path", value = "Method Configuration Namespace"),
+    new ApiImplicitParam(name = "methodConfigurationName", required = true, dataType = "string", paramType = "path", value = "Method Configuration Name")
+  ))
+  @ApiResponses(Array(
+    new ApiResponse(code = 204, message = "Successful Request"),
+    new ApiResponse(code = 404, message = "Workspace or Method Configuration does not exist"),
+    new ApiResponse(code = 500, message = "Rawls Internal Error")
+  ))
+  def deleteMethodConfigurationRoute = cookie("iPlanetDirectoryPro") { securityTokenCookie =>
+    path("workspaces" / Segment / Segment / "methodconfigs" / Segment / Segment) { (workspaceNamespace, workspaceName, methodConfigurationNamespace, methodConfigName) =>
+      delete {
+        requestContext => perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor),
+          WorkspaceService.DeleteMethodConfiguration(workspaceNamespace, workspaceName, methodConfigurationNamespace, methodConfigName))
+      }
+    }
+  }
+
+  @Path("/{configNamespace}/{configName}/rename")
+  @ApiOperation(value = "rename method configuration in a workspace",
+    nickname = "renamemethodconfig",
+    httpMethod = "Post")
+  @ApiImplicitParams(Array(
+    new ApiImplicitParam(name = "workspaceNamespace", required = true, dataType = "string", paramType = "path", value = "Workspace Namespace"),
+    new ApiImplicitParam(name = "workspaceName", required = true, dataType = "string", paramType = "path", value = "Workspace Name"),
+    new ApiImplicitParam(name = "methodConfigurationNamespace", required = true, dataType = "string", paramType = "path", value = "Method Configuration Namespace"),
+    new ApiImplicitParam(name = "methodConfigurationName", required = true, dataType = "string", paramType = "path", value = "Method Configuration Name"),
+    new ApiImplicitParam(name = "newMethodConfigurationName", required = true, dataType = "string", paramType = "path", value = "New Method Configuration Name")
```

Comment:
This is a `body` parameter of type `org.broadinstitute.dsde.rawls.model.MethodConfigurationName`